_summaries/liu2019large-scale_2.md

```diff
@@ -1,19 +1,90 @@
+of this mini batch. 2) Propagation: this step alternatively updates the direct feature v direct and the centroids to minimize the distance between each 
+direct feature and the centroid of its group and maximize the distance to other centroids.
+
+Composing Memory Feature (v-memory): The memory feature relates the centroids in the memory, transferring knowledge to the tail class.
+	
+![LeNet](liu2019large-scale_2a.png)
+												
+Where o belongs  R K is the coefficients hallucinated from the direct feature , o = Thal(v-direct).
+
+Obtaining Dynamic Meta-Embedding: (v-meta) combines the direct feature and the memory feature, and is fed to the classifier for the final class.
+
+		v meta = (1/γ) • (v direct + e X v memory),
+
+where X denotes element-wise multiplication. γ > 0 is seemingly a redundant scalar for the closed-world classification tasks.
+
+# Modulated Attention:
+While dynamic meta-embedding facilitates feature sharing between head and tail classes, it is also vital to discriminate between them. Modulated 
+attention to encourage samples of different classes to use different contexts. a self-attention map SA(f)  is computed from the input feature map by 
+self-correlation . It is used as contextual information and added back  to the original feature map. The modulated attention MA(f) is then designed as 
+conditional spatial attention applied to the self-attention map.The final attention feature map becomes:
+
+![LeNet](liu2019large-scale_2b.png)
+												
+where f is a feature map in CNN, SA(•) is the self attention operation, and MA(•) is a conditional attention function with a softmax normalization.	
+
+Learning: The cosine classifier  is used to produce the final classification results.
+
+Loss Function: The final loss function L is a combination of the cross-entropy classification loss LCE and the large-margin loss between the embeddings 
+and the centroids LLM.
+
+![LeNet](liu2019large-scale_2c.png)
+
+## Experiments: 
+
+• Task : To compute Top-1 classification accuracy.
+
+•Dataset: ImageNet-LT. it has 115.8K images from 1000 categories, with maximally 1280 images per class and minimally 5 images per class. 
+ The additional classes of images in ImageNet-2010 are used as the open set.
+ 
+•Network Architecture: ResNet 10 backbone network for ImageNet-LT.
+
+•Results:The below table represents the performance comparison of different methods
+
+![LeNet](liu2019large-scale_2d.png)
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+•  The  OLTR task  learns from natural long-tail open-end distributed data and optimizes the overall accuracy over a balanced test set.
+
+•  The integrated OLTR algorithm can be used, with dynamic meta-embedding, in order to share visual knowledge between head and tail classes and to reduce 
+   confusion between tail and open classes
+
+•  It gives good accuracy over ImageNet-LT.
```

Comment:
maybe could combine points 2 and 3 and add an additional point pointing out how this paper is the first that sets up datasets for OLTR problem and can serve as a benchmark for future works in OLTR.

Reply:
Done the change as suggest by you.